modules/orca/main.tf

```diff
@@ -142,12 +142,28 @@ module "orca_iam" {
   orca_reports_bucket_name = var.orca_reports_bucket_name
 }
 
+## orca_graphql_0- graphql module that sets up centralized db code
```

Comment:
I would note here that this needs to be run to setup IAM roles needed for GraphQL before the setup of secretmanager so we better understand that. May be good to rename graphql_0 to graphql_iam to better drive the point home too.

Reply:
Added some comments. I am keeping the names deliberately generic to reduce Terraform redeploy messes.